package.json

```diff
@@ -15,19 +15,19 @@
     "test:single": "jest"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.1",
-    "eslint": "^5.10.0",
-    "eslint-plugin-jest": "^22.1.2",
-    "eslint-plugin-mocha": "^5.2.0",
-    "eslint-plugin-react": "^7.10.0",
-    "eslint-plugin-react-native": "^3.1.0",
+    "babel-eslint": "^10.0.2",
+    "eslint": "^6.0.1",
```

Comment:
These changes still work with root-web/myles-mobile, right?

Reply:
Good question.  I am completely unaware of what `root-web/myles-mobile` actually is.  Will have to clone and see if I can spin it up, set the package.json to point to this branch and let CI prove it.   Is there anything else I should be looking at regarding myles?

Reply:
Nope. It's just linting, so assuming the build passes you're probably ok

Reply:
myles-mobile passes the minute I remove the rule that no longer exists (convo in slack).